src/app/content/practiceQuestions/reducer.ts

```diff
@@ -13,6 +13,7 @@ export const initialState: State = {
   isEnabled: false,
   open: false,
   questions: [],
+  questionsAndAnswers: new Map(),
```

Comment:
supposed to keep states as simple objects/arrays, you can convert to a map in the selector if you want